docs/en/siem/detection-engine-intro.asciidoc

```diff
@@ -0,0 +1,97 @@
+[[detection-engine-overview]]
+[role="xpack"]
+
+== Detections (Beta)
+
+beta[]
+
+The SIEM Detections feature automatically searches for threats and creates 
+signals when they are detected. Signal detection rules define the conditions 
+for creating signals. The {siem-app} comes with
+<<prebuilt-rules, prebuilt rules>> that search for suspicious 
+activity on your network and hosts. Additionally, you can
+<<rules-ui-create, create your own rules>>.
+
+[IMPORTANT]
+==============
+See <<detections-permissions, {kib} and index privileges>> required for viewing 
+signals and creating rules.
+==============
+
+[float]
+[[det-engine-terminology]]
+=== Terminology
+
+Signals::
+Always refer to a {siem-soln} produced detections. Signals are never 
+received from third-party systems. When a rule's conditions are met, the
+{siem-app} writes one or more signals to an Elasticsearch `signals` index.
+
+[NOTE]
+==============
+Signal indices are created for each {kib} space. The naming convention is:
+`.siem-signals-<space name>`. For the default space, the signals index is named 
+`.siem-signals-default`.
+==============
+
+Alerts and events::
+Always refer to data the {siem-app} receives from data shippers and external 
+tools, such as Suricata and Elastic Endpoint Security.
+
+[float]
+=== Signals and external alerts
+
+The Detections page displays all signals and alerts. To view signals created 
+by a rule, click *Manage signal detection rules* and then the relevant rule in 
+the All rules table.
+
+To view alerts from external data shippers, click *External alerts*.
+
+[float]
+==== Open and close signals
+
+You can close signals to indicate they do not need any further investigation. 
+By default, the All signals table displays open signals. To view closed 
+signals, click *Closed signals*.
+
+To open and close signals, either:
+
+* Click the *Close/Open signal* icon.
+* Select the signals you want to open or close and then click *Batch actions*
+-> *Close/Open selected*.
+
+[float]
+==== Send signals to the Timeline
+
+To investigate signals in the Timeline, either:
+
+* Click the *View in timeline* icon
+* Select the signals you want to view in the Timeline and then click
+*Batch actions* -> *View selected in timeline*
```

Comment:
As above, it appears `Batch actions` was removed from the `Signals Table` in favor of just having the open/close action in the main Utility Bar.